tests/x11/ghostscript.pm

```diff
@@ -31,31 +31,29 @@ use testapi;
 use utils;
 
 sub run {
-    select_console "x11";
-    x11_start_program('xterm');
+    my $self = shift;
+    $self->select_serial_terminal;
 
-    # become root, disable packagekit and install all needed packages
-    become_root;
+    # disable packagekit and install all needed packages
     quit_packagekit;
     zypper_call "in ghostscript ghostscript-x11";
 
     # special case for gv which is not installed on all flavors
     my $gv_missing = zypper_call("in gv", exitcode => [0, 104]);
-
-    # exit root shell
-    type_string "exit\n";
-
-    my $gs_script = "ghostscript_ps2pdf.sh";
-    my $gs_log    = "ghostscript.log";
-    my $gs_failed = "/tmp/ghostscript_failed";
-    my $reference = "alphabet.pdf";
+    my $gs_script  = "ghostscript_ps2pdf.sh";
+    my $gs_log     = "ghostscript.log";
+    my $gs_failed  = "/tmp/ghostscript_failed";
+    my $reference  = "alphabet.pdf";
 
     # download ghostscript converter script and test if download succeeded
+    # switch to right directory and create files which can be accessed by test user
+    script_run "cd /home/$username";
     assert_script_run "wget " . data_url("ghostscript/$gs_script");
     assert_script_run "test -s $gs_script";
 
     # convert example *.ps files to *.pdf
     assert_script_run "sh ./$gs_script";
+    # assert_script_run "cp alphabet.pdf /tmp";
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

Reply:
I forgot to remove this, that was only a test.